cmake/Modules_CUDA_fix/upstream/FindCUDAToolkit.cmake

```diff
@@ -0,0 +1,1105 @@
+# Distributed under the OSI-approved BSD 3-Clause License.  See accompanying
```

Comment:
This is a second `FindCUDAToolkit.cmake`. We should only need one.

Reply:
I should cover it

Reply:
1. The plan is to delete `Modules_CUDA_fix` so I don't think it's good to move this.
2. `FindPackageHandleStandardArgs` is a standard CMake module, so your justification doesn't make sense to me.

Reply:
1. I am agree with deleting Modules_CUDA_fix
2. But currently I have to put FindCUDAToolkit.cmake on Modules_CUDA_fix because it contains "include(${CMAKE_CURRENT_LIST_DIR}/FindPackageHandleStandardArgs.cmake)" on line 868. I am not sure why upstream module use a hard-wired path to FindPackageHandleStandardArgs.cmake, but if we just want to use it without changes, we have to also copy FindPackageHandleStandardArgs.cmake to Modules.

Reply:
OK, revert to your variant